packages/trimerge-sync-indexed-db/src/IndexedDbCommitRepository.ts

```diff
@@ -131,7 +133,7 @@ export class IndexedDbCommitRepository<CommitMetadata, Delta, Presence>
     this.addStoreMetadata = addStoreMetadata;
 
     const dbName = getDatabaseName(docId);
-    console.log(`[TRIMERGE-SYNC] new IndexedDbBackend(${dbName})`);
+    this.logger.log(`[TRIMERGE-SYNC] new IndexedDbBackend(${dbName})`);
```

Comment:
one thing to consider is making logger optional and using the syntax:
```suggestion
    this.logger?.log(`[TRIMERGE-SYNC] new IndexedDbBackend(${dbName})`);
```

maybe even default to no logger?

Reply:
Yeah, I like that. I was thinking it might be surprising if it didn't log by default but I guess if someone is just consuming these apis, they probably wouldn't be expecting this to generate log statements by default.